chart/skywalking/templates/oap-clusterrole.yaml

```diff
@@ -25,7 +25,7 @@ metadata:
     heritage: "{{ .Release.Service }}"
 rules:
 - apiGroups: [""]
-  resources: ["pods", "pods/log", "endpoints", "services", "nodes"]
+  resources: ["pods", "pods/log", "endpoints", "services", "nodes", "namespaces"]
```

Comment:
Which feature in SkyWalking requires this permission?

Reply:
![image](https://user-images.githubusercontent.com/37059529/226817673-3db9337f-42d0-4db6-9ca9-071fd913d4cb.png)